man/sd_device_enumerator_add_match_parent.xml

```diff
@@ -0,0 +1,220 @@
+
+    <funcprototype>
+      <funcdef>int <function>sd_device_enumerator_add_nomatch_sysname</function></funcdef>
+      <paramdef>sd_device_enumerator *<parameter>enumerator</parameter></paramdef>
+      <paramdef>const char *<parameter>sysname</parameter></paramdef>
+    </funcprototype>
+
+    <funcprototype>
+      <funcdef>int <function>sd_device_enumerator_add_match_tag</function></funcdef>
+      <paramdef>sd_device_enumerator *<parameter>enumerator</parameter></paramdef>
+      <paramdef>const char *<parameter>tag</parameter></paramdef>
+    </funcprototype>
+
+    <funcprototype>
+      <funcdef>int <function>sd_device_enumerator_allow_uninitialized</function></funcdef>
+      <paramdef>sd_device_enumerator *<parameter>enumerator</parameter></paramdef>
+    </funcprototype>
+
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+    <para>The <function>sd_device_enumerator_add_match_parent()</function> function adds a filter to
+    the <parameter>enumerator</parameter> so that only devices under the tree of the specified
+    <parameter>parent</parameter> device are enumerated. If this is called multiple times the
+    previously set <parameter>parent</parameter> device is cleared.</para>
+
+    <para>The <function>sd_device_enumerator_add_all_parents()</function> function adds a filter to the
+    <parameter>enumerator</parameter> so that parent devices of the matching devices are
+    also included in the enumeration. This is useful for cases where you want to include all parent
+    devices in the enumeration, such as when you are interested in the entire device tree leading up
+    to a specific device.</para>
+
+    <para>The <function>sd_device_enumerator_add_match_property()</function> function adds a filter to
+    the <parameter>enumerator</parameter> so that only devices with the specified
+    <parameter>property</parameter> equals to the<parameter>value</parameter> are enumerated.
+    Both <parameter>property</parameter> and <parameter>value</parameter> can be a glob pattern.
+    When this is called multiple times, then devices that have at least one of the specified
+    properties with matching values are enumerated. That is, filters are ORed.q</para>
+
+    <para>The <function>sd_device_enumerator_add_match_property_required()</function> function adds a
+    filter to the <parameter>enumerator</parameter> so that only devices with the specified
+    <parameter>property</parameter> equals to the<parameter>value</parameter> are enumerated. This
+    function is similar to <function>sd_device_enumerator_add_match_property</function>, but it
+    requires that the property must exist on the device, not just match a value.
+    When this is called multiple times, devices that have all specified properties with matching
+    values are enumerated. That is, filters are ANDed.</para>
+
+    <para>The <function>sd_device_enumerator_add_match_subsystem()</function>functions enumerates all
```

Comment:
- missing space after `</function>`
- `functions` -> `function`
- strictly speaking, this does not enumerate, but adds a filter...